terraform/modules/base_cluster/traefik.tf

```diff
@@ -1,6 +1,6 @@
 resource "helm_release" "traefik" {
   name       = "traefik"
-  repository = "https://kubernetes-charts.storage.googleapis.com"
+  repository = "https://charts.helm.sh/stable"
```

Comment:
I think the helm charts repo is now [deprecated](https://helm.sh/blog/charts-repo-deprecation/) since november.
Do we want to switch repos for traefik/prometheus/grafana at some point so we can keep on getting updates?

Reply:
We definitely should switch at some point, but I think that should happen as part of a different PR just so we're changing fewer things at once